scipy/stats/_multivariate.py

```diff
@@ -3850,3 +3852,455 @@ def rvs(self, dim, size=1, random_state=None):
+    ``logpmf(x, m, n)``
+        Log of the probability mass function.
+    ``rvs(m, n, size=1, random_state=None)``
+        Draw random samples from a multivariate hypergeometric
+        distribution.
+    ``mean(m, n)``
+        Mean of the multivariate hypergeometric distribution.
+    ``var(m, n)``
+        Variance of the multivariate hypergeometric distribution.
+    ``cov(m, n)``
+        Compute the covariance matrix of the multivariate
+        hypergeometric distribution.
+
+    Parameters
+    ----------
+    %(_doc_default_callparams)s
+    %(_doc_random_state)s
+
+    Notes
+    -----
+    %(_doc_callparams_note)s
+
+    The probability mass function for `multivariate_hypergeom` is
+
+    .. math::
+
+        P(X_1 = x_1, X_2 = x_2, \ldots, X_k = x_k) = \frac{\binom{m_1}{x_1}
+        \binom{m_2}{x_2} \cdots \binom{m_k}{x_k}}{\binom{M}{n}}, \\ \quad
+        (x_1, x_2, \ldots, x_k) \in \mathbb{N}^k \text{ with }
+        \sum_{i=1}^k x_i = n
+
+    where :math:`m_i` are the number of objects of type :math:`i`, :math:`M`
+    is the total number of objects in the population (sum of all the
+    :math:`m_i`), and :math:`n` is the size of the sample to be taken
+    from the population.
+
+    .. versionadded:: 1.6.0
+
+    Examples
+    --------
+
+    To evaluate the probability mass function of the multivariate
+    hypergeometric distribution, with a dichotomous population of size
+    :math:`10` and :math:`20`, at a sample of size :math:`12` with
+    :math:`8` objects of the first type and :math:`4` objects of the
+    second type, use:
+
+    >>> from scipy.stats import multivariate_hypergeom
+    >>> multivariate_hypergeom.pmf(x=[8, 4], m=[10, 20], n=12)
+    0.0025207176631464523
+
+    The `multivariate_hypergeom` distribution is identical to the
+    corresponding `hypergeom` distribution (tiny numerical differences
+    notwithstanding) when only two types (good and bad) of objects
+    are present in the population as in the example above. Consider
+    another example for a comparison with the hypergeometric distribution:
+
+    >>> from scipy.stats import hypergeom
+    >>> multivariate_hypergeom.pmf(x=[3, 1], m=[10, 5], n=4)
+    0.4395604395604395
+    >>> hypergeom.pmf(k=3, M=15, n=4, N=10)
+    0.43956043956044005
+
+    The functions ``pmf``, ``logpmf``, ``mean``, ``var``, ``cov``, and ``rvs``
+    support broadcasting, under the convention that the vector parameters
+    (``x``, ``m``, and ``n``) are interpreted as if each row along the last
+    axis is a single object. For instance:
+
+    >>> multivariate_hypergeom.pmf(x=[[8, 4], [3, 1]], m=[[10, 20], [10, 5]], n=[12, 4])
+    array([0.00252072, 0.43956044])
+
+    This combines the result of previous two examples in a single array.
+    In this example, the last axis of the each array ``x``, ``m``, and ``n``
+    are treated as independent objects and the ``pmf`` is exaluated over
+    each of them to generate a combined result.
```

Comment:
> ```the last axis of the each array ``x``, ``m``, and ``n`` are treated as independent objects ```

Suppose I want to understand what this sentence says about just `x`. Does it say:

> ```the last axis of ``x`` is treated as an independent object.```

?

Reply:
Yes. It says that.

Reply:
Hmm, are you sure that sentence literally means what you intend it to mean? I understand phrases like "treat each _row_ as an object" or "treat each `x[i]`" as an object, but "treat the last axis as an object"? An axis of an array is a concept, not a collection of elements of the array.

Reply:
I may have stated it wrong but you got what I mean. I will rephrase it as you have suggested. It sounds much better that way. Thanks!

> "treat each `x[i]`"

I think you mean "treat each `x[..., i]`" as there could be more than 3-dimensional arrays as input.

Reply:
I don't think so. At least for two dimensions, it seems to treat each `x[i]`, that is, each `x[i, :]` as an object that corresponds with each `m[i, :]` and each `n[i]`